eslint.config.js

```diff
@@ -33,6 +33,7 @@ export default tseslint.config(
           argsIgnorePattern: "^_",
         },
       ],
+      "@typescript-eslint/no-explicit-any": "warn",
```

Comment:
We don't want either warnings or errors in the compilation output. You should disable this rule.